datahub-web-react/src/app/entity/shared/Properties.tsx

```diff
@@ -1,16 +1,19 @@
 import { Space, Table, Typography } from 'antd';
 import React from 'react';
+import { ColumnsType } from 'antd/es/table';
 import { StringMapEntry } from '../../../types.generated';
 
 export type Props = {
     properties: StringMapEntry[];
 };
 
 export function Properties({ properties }: Props) {
-    const propertyTableColumns = [
+    const propertyTableColumns: ColumnsType<StringMapEntry> = [
         {
             title: 'Name',
             dataIndex: 'key',
+            sorter: (a, b) => b.key.localeCompare(a.key),
+            defaultSortOrder: 'descend',
```

Comment:
@RickardCardell : is the intent to have the default display order be a-z? or z-a?

Reply:
Nice catch. Intention is 'a-z'. I've pushed a fix